content/top-5-programming-language-for-backend-web-programmers.md

```diff
@@ -0,0 +1,1333 @@
+---
+title: Top 5 Programming Language for Serious Backend Programmer
+desc: Top 5 Programming Language for Serious Backend Programmer
+author: Valian Masdani
+github: vmasdani
+twitter: VMasdani
+telegram: vmasdani
+cover: https://content.techgig.com/thumb/msid-67337479,width-860,resizemode-4/Top-5-programming-languages-for-backend-web-development.jpg?94297
+date: 2022-02-05
+categories:
+  - backend
+  - web
+  - programming
+---
+
+There are tons upon tons of programming languages in the market nowadays, and it might be overwhelming for people who just started programming. This question is often asked among the beginner programmers:
+
+**How do we pick the right programming language?**
+
+Here's the thing about picking the most powerful, best ever programming language:
+
+1. You can't
+2. It depends
+
+Why **can't you**? Because there are always a few considerations in picking a programming language for a certain field of work/expertise. In my experience, some of them includes:
+
+1. **Popularity**  
+   Is the language commonly used? Does it have big communities? Are big companies using it, supporting it, and endorsing it?  
+   Popularity should be a strong factor in deciding which tech to use, because the bigger the community is, the easier for you to find a solution to a sometimes weird and quirky problem in a programming language, compiler, tooling, etc. You don't want to spend a few days trying to fix a weird bug when the fix is actually just adding a semicolon or comma, adding a single configuration line, or simple stupid things that can cost you precious time when trying to fix it.
```

Comment:
```suggestion
   Popularity should be a strong factor in deciding which tech to use, because the larger the community is, the easier for you to find a solution to a sometimes weird and quirky problem in a programming language, compiler, tooling, etc. You don't want to spend a few days trying to fix a weird bug when the fix is actually just adding a semicolon or comma, adding a single configuration line, or simple stupid things that can cost you precious time when trying to fix it.
```